Extensions/Firebase/JsExtension.js

```diff
@@ -0,0 +1,1715 @@
+// @flow
+/*::
+import { type Trans } from '../../newIDE/app/node_modules/@lingui/macro'
+*/
+
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function (
+      _ /*: Trans */, 
+      gd /*: libGDevelop */
+    ) {
+    const extension /*: gdPlatformExtension */ = new gd.PlatformExtension();
```

Comment:
This extra typing is not necessary, because `gd` is typed. Keep it like the other extensions:
`const extension = new gd.PlatformExtension();`